### PR TITLE
Telescience Restoration - Should Work This Time!

### PR DESCRIPTION
**Pull request by BlueNothing**
Restores the functionality of Telescience to all stations by adding the telepad and control console to the techweb as a node after Bluespace Travel and Miniaturized Bluespace. Now modularized!
/🆑

[why]: Telescience is a famous and well-loved part of the SS13 experience that has been replicated only in hollow fashion in recent days with devices like the quantum pad (which still has a useful niche if you want A->B->C->A transportation, or A->B->A transportation without a second person involved), the bluespace launchpad (which mostly everyone treats as useless, because its range means that it rarely gets you anywhere interesting unless you're already somewhere at least almost as interesting, but which does have a much simpler control scheme), or the traitor briefcase launchpad (which is still somewhat useful, if only due to being portable and letting you yoink things from another room (pairing well with x-ray or thermal vision)). A proper restoration would likely be no shortage of amusement to many.

Should also close #9433

### CODE REVIEW

**Comment by ghost** [comment]
nice +1

File: code/modules/shuttle/supply.dm [hunk 1]
```diff
@@ -25,7 +25,9 @@ GLOBAL_LIST_INIT(blacklisted_cargo_types, typecacheof(list(
 		/obj/item/shared_storage,
 		/obj/structure/extraction_point,
 		/obj/machinery/syndicatebomb,
-		/obj/item/hilbertshotel
+		/obj/item/hilbertshotel,
+		/obj/machinery/telepad, //Hippie - Shouldn't be exportable, could be used to retrieve someone from the station if it was on the Centcom Z-level.
+        /obj/machinery/computer/telescience //Hippie - Used to control the telepad.
```
Comment (k-k-karma) [inline code/modules/shuttle/supply.dm]: possibly add to the list via init proc

**Comment by BlueNothing** [comment]
How does one do that?

**Comment by yobob591** [comment]
owo +1

**Comment by Lxtoofonea** [comment]
Yee +1

**Comment by Manly-Dwarf** [comment]
I'm still very much against Telescience returning due to its minimal costs, how easy it is to abuse, and how it's even more Science powergame.
Circuits are obnoxious and insanely overpowered (and have recently been buffed for some godforsaken reason), Nanites have antistuns and healing, there's the bombs which have mysteriously been changed way too much as of late, xenobiology, robotics, the weapons
Just stop. Science already has so many fucking tools that it's going to make me puke, and almost all of them are extremely powerful if you know how to make them work.

You'd have to really find a way to justify this other than "the playerbase wants it", because this ain't some harmless goat plushie, this is something that has been historically abused to hell and back. Also the playerbase apparently likes Shadowlings, so...

**Comment by Hate9** [comment]
Circuits can do very little harm whatsoever (and IIRC the thrower circuits still do no damage).
Nanites can outright kill you if you get even a minor electric shock (and the chances of it happening increase as the shocks get larger).
For that matter, science can't make most of its weapons without security's help, because the console in R&D can't print them. The only weapons they can make without that help come from hacking the autolathe.
Xenobiology requires a lot of work to get anything fun, and robotics requires people to make into robots, or a metric fuck-tonn of materials from mining if they want to make mechs.
I can't comment on bombs, since I basically never do them, but aren't they also an Atmospherics thing?
EDIT: Also, anything that's part of techwebs (so basically all of science) requires you spend and wait a considerable amount of time before unlocking it.

**Comment by BlueNothing** [comment]
Um... throwers can do damage.

**Comment by BlueNothing** [comment]
Sci *can* make weapons, but need an emag or mind shields or sec-donated
firing pins to fire them.

**Comment by Hate9** [comment]
Oh, did that get changed back? Nevermind on that point, then.

**Comment by BlueNothing** [comment]
Bombs are science and atmos, yes, but doing them as atmos requires a clever
trick or canister charges and running like hell.

**Comment by Hate9** [comment]
> Bombs are science and atmos, yes, but doing them as atmos requires a clever trick or canister charges and running like hell.

Or theft.

**Comment by Manly-Dwarf** [comment]
>Circuits can do very little harm whatsoever (and IIRC the thrower circuits still do no damage).

Have you been on the server, at all, in the past... say, five months?

**Comment by Hate9** [comment]
> Have you been on the server, at all, in the past... say, five months?

Yes. Yes I have.

**Comment by ghost** [comment]
@Hate9 I agree with KBB that that almost everything you said was wrong. Except xenobiology. Lot of work and no reward is the name of the game there...

**Comment by Hate9** [comment]
@HideAndSeekLOGIC where did he say that? All we seemed to disagree on was that throwers can damage things (and I haven't noticed anyone making thrower weapons, so I assumed they were still nerfed).

**Comment by bardofrage** [comment]
+1

**Pull request merged**